### ISSUE TITLE
Use original react-native-swiper

### ISSUE THREAD

**Issue by ghost** [FuYaoDe/react-native-app-intro#97]
Instead of @FuYaoDe fork that is not being maintained.

**Comment by aforty** [comment]
I have this working on my branch if you're interested.

`yarn add git+https://github.com/aforty/react-native-app-intro\#fix-swiper-dependency`

**Comment by michaelforrest** [comment]
Yes this component now crashes when I try to use it in a new app due to this dependency.

**Comment by ghost** [comment]
@michaelforrest I've reached ou to @FuYaoDe by email to see if I can help with the maintenance.
